comfyui_custom_scripts/extensions/webui_checkpoint_loader.js

```diff
@@ -0,0 +1,97 @@
+import { app } from "/scripts/app.js";
+
+
+function addHiddenMultilineWidget(node, name, app) {
+	const widget = {
+		type: "customtext",
+		name,
+		get value() {
+			return this.inputEl.value;
+		},
+		set value(x) {
+			this.inputEl.value = x;
+		},
+		draw: function (ctx, _, widgetWidth, y, widgetHeight) {},
+	};
+	widget.inputEl = document.createElement("textarea");
+	widget.inputEl.classList.add('hidden-multiline-input');
+	widget.inputEl.style.display = 'none';
+	widget.inputEl.value = '';
+	widget.inputEl.placeholder = '';
+	widget.parent = node;
+	document.body.appendChild(widget.inputEl);
+
+	node.addCustomWidget(widget);
+
+	node.onRemoved = function () {
+		// When removing this node we need to remove the input from the DOM
+		for (let y in this.widgets) {
+			if (this.widgets[y].inputEl) {
+				this.widgets[y].inputEl.remove();
+			}
+		}
+	};
+
+	return { minWidth: 400, minHeight: 200, widget };
+}
+
+
+const ext = {
+	// Unique name for the extension
+	name: "Test",
```

Comment:
should we rename this to something more meaningful?

Reply:
ah my bad, that's a leftover